packages/app/src/CacheManager.ts

```diff
@@ -34,7 +33,7 @@ export default class CacheManager implements EstimatorCache {
     const { GCS } = this.cacheModes
     let estimates
 
-    switch (this.currentCacheMode) {
+    switch (configLoader().CACHE_MODE) {
```

Comment:
Similar thoughts as below in the `EstimatorCacheGoogleCloudStorage.ts` file

Reply:
Same reason as above ☝️